lib/iris/fileformats/netcdf/loader.py

```diff
@@ -719,18 +719,19 @@ def set(
             Kwargs specifying chunksizes for dimensions of file variables.
             Each key-value pair defines a chunksize for a named file
             dimension, e.g. {'time': 10, 'model_levels':1}.
+            Values of -1 will lock the chunk to the size of it's shape.
 
         Notes
         -----
         This function acts as a contextmanager, for use in a 'with' block.
 
-        Example:
-
-        #todo
-        # >>> from iris.fileformats.netcdf.loader import CHUNK_CONTROL
-        # >>> from iris import sample_data_path
-        # >>> with CHUNK_CONTROL.set('var1', model_level=1, time=50):
-        # ...     cubes = iris.load(sample_data_path("toa_brightness_stereographic.nc"))
+        >>> import iris
+        >>> from iris.fileformats.netcdf.loader import CHUNK_CONTROL
+        >>> with CHUNK_CONTROL.set("air_temperature", time=180, latitude=-1):
+        ...     cube = iris.load(iris.sample_data_path("E1_north_america.nc"))[0]
+        >>>
+        >>> print(cube.lazy_data().chunksize)
+        (180, 37, 49)
```

Comment:
I can appreciate that this is here as a demonstration, but it doesn't demonstrate much without showing what `chunksize` would be without modification. I'd say either:

1. Add an unmodified `load()` call
2. Remove the demonstration and just stick with the example usage

Reply:
The latter makes more sense; it's meant to be show be an example of use, not an example of output.